test/converters/xcresult3/model3/testresults.go

```diff
@@ -0,0 +1,63 @@
+package model3
+
+type TestNodeType string
+
+// These are all the types the xcresulttool (version 23500, format version 3.53) supports.
```

Comment:
Maybe as an improvement, we should put together a test, that checks if a new format was shipped with a new Xcode version.